common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -462,7 +462,6 @@ def test_account_settings_a11y(self):
         self.account_settings_page.a11y_audit.config.set_rules({
             'ignore': [
                 'link-href',  # TODO: AC-233, AC-238
-                'skip-link',  # TODO: AC-179
```

Comment:
@clrux  Removing these 'skip-link' lines will result in a check for the skip link on the page during the test, so this should provide the test coverage needed.